subsys/caf/modules/sensor_manager.c

```diff
@@ -180,7 +180,11 @@ static void sensor_wake_up_post(const struct sm_sensor_config *sc, struct sensor
 
 static void trigger_handler(const struct device *dev, const struct sensor_trigger *trigger)
 {
-	sensor_trigger_set(dev, trigger, NULL);
+	int ret = sensor_trigger_set(dev, trigger, NULL);
+
+	if (ret < 0) {
+		LOG_ERR("Cannot activate a sensor trigger (%d)", ret);
```

Comment:
If this cannot happen (and it seems we really expect it cannot happen), let's use assert instead (also in the other case)

btw - you can check the doc or implementation of trigger set to see if this will fail on any malfunction or returns error only if improperly called (in this other case assert is ok)